sktime/forecasting/model_selection/_split.py

```diff
@@ -126,6 +126,28 @@ def get_cutoffs(self, y=None):
         return check_cutoffs(self.cutoffs)
 
 
+class OnlineSplitter(BaseSplitter):
```

Comment:
This seems to be the same as `SlidingWindowSplitter(start_with_window=True, window_length=1, fh=1)`

Reply:
You are totally right, I'll remove it